### ISSUE TITLE
managed-symbols-available fails against vmr output on .NET 8

### ISSUE THREAD

**Issue by tmds** [redhat-developer/dotnet-regular-tests#267]
We've excluded the managed symbols from previous .NET releases since they were causing UX issues: https://github.com/dotnet/source-build/issues/2623.

https://github.com/redhat-developer/dotnet-regular-tests/pull/262 updates the test to expect the managed symbols for .NET 8.

This test is currently passing with rpms, but not with the vmr tarball output.

@omajid does including managed symbols no longer cause UX issues with .NET 8?

**Comment by omajid** [comment]
I was told that sourcelink now correctly points to git repositories (instead of non-existent local paths). That should result in tools like vscode not behaving badly when loading managed symbols. I haven't verified this myself.

**Comment by tmds** [comment]
If the symbols are working again, probably it makes sense for the vmr output to include them again by default?

**Comment by omajid** [comment]
The VMR build produces symbols, they are just put in a separate tarball, not the main SDK tarball. AFAIK, that has always been the case. When we stopped shipping symbols, what we actually did was just [stop extracting the contents of the symbol archive](https://src.fedoraproject.org/rpms/dotnet6.0/blame/dotnet6.0.spec?identifier=ecf6e43a6ba8765b040da9cf93575908a55bc298#_560-564).

If we are testing the just-built SDK, we should probably extract the symbol archive into the extracted-SDK location too.
